core/storage/user/gae_models_test.py

```diff
@@ -189,8 +274,8 @@ def test_export_data_for_nonexistent_user_raises_exception(self) -> None:
             user_models.UserSettingsModel.export_data('fake_user')
 
     def test_export_data_for_trivial_case_returns_data_correctly(self) -> None:
-        user = user_models.UserSettingsModel.get_by_id(self.USER_1_ID)
-        user_data = user.export_data(user.id)
+        user_object = user_models.UserSettingsModel.get_by_id(self.USER_1_ID)
```

Comment:
This should probably be user_model, since it's an item at the storage layer rather than the domain layer. Ditto elsewhere.

(Good spot on seeing that it shouldn't be just "user", though!)